tests/integration/api/test_azure_integration.py

```diff
@@ -0,0 +1,81 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
+APP_KEY = os.environ.get("DD_TEST_CLIENT_APP_KEY", "a" * 40)
+API_HOST = os.environ.get("DATADOG_HOST")
+
+
+class TestAzureIntegration:
+
+    test_tenant_name = "testc44-1234-5678-9101-cc00736ftest"
+    test_client_id = "testc7f6-1234-5678-9101-3fcbf464test"
+    test_client_secret = "testingx./Sw*g/Y33t..R1cH+hScMDt"
+    test_new_tenant_name = "1234abcd-1234-5678-9101-abcd1234abcd"
+    test_new_client_id = "abcd1234-5678-1234-5678-1234abcd5678"
+    not_yet_installed_error = 'Azure Integration not yet installed.'
+
+    @classmethod
+    def setup_class(cls):
+        initialize(api_key=API_KEY, app_key=APP_KEY, api_host=API_HOST)
+
+    def test_azure_create(self):
```

Comment:
Same comment here. It's fine to combine all tests under one `test_azure_crud` so that we can rely on the order of execution of operations.